h2o-algos/src/main/java/hex/modelselection/ModelSelectionModel.java

```diff
@@ -84,8 +84,9 @@ public static class ModelSelectionParameters extends Model.Parameters {
         public double _obj_reg = -1.0;
         public double[] _lambda = new double[]{0.0};
         public boolean _use_all_factor_levels = false;
-        public boolean _build_glm_model = true;
+        public boolean _build_glm_model = false;
```

Comment:
Tomas:  This parameter is just for our one customer.  If this parameter is set to true, it will return a GLM model associated with the best predictor subset.  If it is set to false, it will not return the GLM model, it will just return the predictor subset names and the coefficients associated with the predictors.  

Talked to them and they don't care about getting the GLM model but rather wants things to run fast.  Hence, the change.